.github/workflows/reusable_provenance.yaml

```diff
@@ -24,7 +24,10 @@ on:
 jobs:
   get_inputs:
     outputs:
-      artifact-path: ${{ steps.artifact-path.outputs.name }}
+      # Resolves to one or more files which are passed to the SLSA provenance generator.
```

Comment:
It looks like the code further down assumes this is only one file, not more than one.

Reply:
Thanks, done.